src/reducers/selection_reducer.js

```diff
@@ -0,0 +1,17 @@
+import _ from 'lodash';
+import { select } from '../action_types.js';
+
+export default (state=[], action)=>{
+
+  if(action.type === select){
```

Comment:
Add some spaces to this code, let it breath. Also as a general rule I like to make sure to break up my reducers into functions. Here's an example of what I mean. https://github.com/trabian/ensenta-ezadmin/blob/5e413271b77b959c916e28b717f20b6aa7f410fb/src/components/grid/reducer.js#L230-L255